### DISCUSSION TITLE
Função

### DISCUSSION THREAD

**Discussion by SabrinaFurtado** [dgadelha/Portugol-Webstudio#147] [Ideias e sugestões]
Preciso criar uma função capaz de processar um número e verificar se é superior a 10. O resultado verdadeiro (verdadeiro) deve ser emitido para o console se o número for superior. Caso não seja, o resultado deve ser falso (falso). 
No programa abaixo da erro de compilação - Expressão inválida: numero (linha 3, posição 20)
o que seria? qual meu erro?
Código abaixo:

programa
{
    funcao verificar(numero)
    {
        se (numero > 10)
        {
            escreva("verdadeiro")
        }
        senao
        {
            escreva("falso")
        }
    }

    funcao inicio()
    {
        verificar(5)
        verificar(10)
        verificar(15)
    }
}

**Comment by dgadelha** [comment]
Olá,

Você precisa definir o tipo do parâmetro

`funcao verificar(inteiro numero)`
